app/models/embedded_ansible_worker/object_management.rb

```diff
@@ -53,4 +54,74 @@ def ensure_host(provider, connection)
       :variables => {'ansible_connection' => "local"}.to_yaml
     ).id
   end
+
+  def ensure_plugin_playbooks_project_seeded(connection)
+    clean_consolidated_plugin_directory
+    copy_plugin_ansible_content
+
+    commit_git_plugin_content
+
+    if project = existing_plugin_playbook_project(connection)
+      update_playbook_project(project)
+    else
+      create_playbook_project(connection)
+    end
+  ensure
+    # we already have 2 copies: one in the gem and one imported into ansible in the project, delete the temporary one
+    clean_consolidated_plugin_directory
+  end
+
+  private
+
+  CONSOLIDATED_PLUGIN_PLAYBOOKS_TEMPDIR = Pathname.new("/var/lib/awx_consolidated_source/projects/gem_ansible_content").freeze
+  def clean_consolidated_plugin_directory
+    FileUtils.rm_rf(CONSOLIDATED_PLUGIN_PLAYBOOKS_TEMPDIR)
+  end
+
+  def copy_plugin_ansible_content
+    FileUtils.mkdir_p(CONSOLIDATED_PLUGIN_PLAYBOOKS_TEMPDIR)
+
+    #TODO: make this a public api via an attr_reader
+    Vmdb::Plugins.instance.instance_variable_get(:@registered_ansible_content).each do |content|
+      FileUtils.cp_r(Dir.glob("#{content.path}/*"), CONSOLIDATED_PLUGIN_PLAYBOOKS_TEMPDIR)
+    end
+  end
+
+  def commit_git_plugin_content
+    Dir.chdir(CONSOLIDATED_PLUGIN_PLAYBOOKS_TEMPDIR) do
+      require 'rugged'
+      repo = Rugged::Repository.init_at(".")
+      index = repo.index
+      index.add_all("*")
+      index.write
+
+      options              = {}
+      options[:tree]       = index.write_tree(repo)
+      options[:author]     = options[:committer] = { :email => "user@example.com", :name => 'Author', :time => Time.now }
+      options[:message]    = "YOLO Initial Commit"
+      options[:parents]    = []
+      options[:update_ref] = 'HEAD'
+      Rugged::Commit.create(repo, options)
+    end
+  end
+
+  PLUGIN_PLAYBOOK_PROJECT_NAME = "Default ManageIQ Playbook Project".freeze
```

Comment:
Does this get displayed in the UI? If so we should use the product name.

Reply:
@syncrou @gmcculloug Yeah, I have no idea.  This was me just naming it knowing it would change.

Reply:
@jrafanie, @carbonin - Yes, This name would show up in the services 'repo' list, also in the Automate inline method 'repo' list.  So we would want a descriptive name that makes sense to parties expecting to find their seeded playbooks.

Reply:
@syncrou @gmcculloug naming is hard.  Let me know what name you want.  🏃 🚌

Reply:
I would stick with the convention from the other objects so something like `"#{I18n.t("product.name")} Default Project"`